frank-doc-doclet/src/main/java/org/frankframework/frankdoc/doclet/Doclet.java

```diff
@@ -54,6 +54,7 @@ class Doclet {
 				FrankElementFilters.getExcludeFilter(), FrankElementFilters.getExcludeFiltersForSuperclass());
 
 			model = FrankDocModel.populate(options.getDigesterRulesUrl(), options.getRootClass(), repository);
+			log.info("Found classes: {}", repository.size());
```

Comment:
```suggestion
			log.info("Found classes: {}", repository::size);
```

Reply:
If I do that, I get the warning: `'org.apache.logging.log4j.util.Supplier' is deprecated`, so I'll leave it for now.